src/identity/apis/auth.ts

```diff
@@ -227,6 +228,25 @@ export const fetchAccountDetails = async (
   return accountDetails
 }
 
+// change the user's default account
+export const putDefaultQuartzAccount = async (
```

Comment:
Not a huge deal, but we might consider calling this `updateDefaultQuartzAccount` since `PUT` is an obvious HTTP verb, and `update` is a little more descriptive, and this is the layer where we should feel comfortable translating HTTP concepts into app-domain concepts. In other words PUT is an HTTP-level concept, `update` is an app-domain concept.

For example, if we decide that we actually don't want to use PUT, and instead the implementation changes to POST, this method will either make no sense, or need to be changed in lockstep

Reply:
Makes sense, and agree. I've changed this to `updateDefaultQuartzAccount` throughout.